pipeline-runner/R/seurat-3-upload_seurat_to_aws.R

```diff
@@ -78,17 +78,18 @@ upload_seurat_to_aws <- function(input, pipeline_config, prev_out) {
 }
 
 find_cluster_columns <- function(scdata) {
+  meta <- scdata@meta.data
 
   # exclude all group columns, including duplicates
-  group_cols <- find_group_columns(scdata@meta.data, remove.dups = FALSE)
+  group_cols <- find_group_columns(meta, remove.dups = FALSE)
+  scdblfinder_cols <- grep('^scDblFinder', colnames(meta), value = TRUE)
```

Comment:
why not add it to `exclude_cols`? don't really feel strongly about it though. whatever you think looks best is fine by me.

Reply:
That's what I did initially as well but it interferes with how `exclude_cols` is used further down (checking to see if a column is sample as `samples` or other group column). I'll rename `exclude_cols` to make this more obvious